core/ReleaseNotes.md

```diff
@@ -1,6 +1,83 @@
 Release Notes
 =============
 
+OpenRocket 22.xx Beta 1
```

Comment:
note, we'll need to make sure that the [build version](https://github.com/openrocket/openrocket/blob/unstable/core/resources/build.properties#L3) is updated appropriately to ensure that these are consistent.

Reply:
Yes.  I don't really know how we're going to track the builds as we go through the beta, or even how often we will be releasing new builds.

Release notes built into the app would be nice, but I'm not about to ask for that now. :)